### ISSUE TITLE
animated bio

### ISSUE THREAD

**Issue by ooflord20002** [zigzatuzoo/Rec-Room-API-Tool-Box#2]
i know this tool was posted 2 years ago, but it doesnt work at all. the page it goes to to update the bio, doesnt even exist. it cant update

**Comment by zigzatuzoo** [comment]
Yes as of now we can't necessarily promise anything we make now will work next week. Rec room is in the process of closing a lot of public endpoints unfortunately, I do apologize, once we get some more info on the matter from rec room we may proceed to update and make more tools but until then a lot of us are at the will of RR.
